src/components/DocumentContent.vue

```diff
@@ -374,17 +410,27 @@ export default {
       <hook name="document.content.ner:after" class="d-flex flex-row justify-content-end align-items-center"></hook>
     </div>
 
-    <hook name="document.content.body:before"></hook>
-    <document-content-slices
-      ref="slices"
-      :bufferize-all="!!localSearchOccurrences"
-      :class="{ 'document-content__body--rtl': isRightToLeft }"
-      :slices="virtualContentSlices"
-      class="document-content__body container-fluid py-3"
-      @placeholder-visible="onContentSlicePlaceholderVisible"
-    />
-    <hook name="document.content.body:after"></hook>
-
+    <div class="border shadow-sm m-3">
+      <div v-if="nbPages > 1" class="p-1 bg-lighter">
+        <b-pagination
+          v-model="page"
+          align="center"
+          class="m-0"
+          size="sm"
+          :per-page="1"
+          :total-rows="nbPages"
+        ></b-pagination>
+      </div>
+      <b-overlay :show="$wait.is('loadContentSlice')" opacity="0.6" rounded spinner-small>
+        <hook name="document.content.body:before"></hook>
```

Comment:
@pirhoo Should this hook `document.content.body:before` go before the pagination ? (L412) (same question for `document.content.body:after`

Reply:
Good question: this hook might be used to add elements "around" the pagination block so I'm tempted to move it just outside of the `b-overlay`